src/app/layout/component/user-settings-dialog/user-settings-dialog.component.spec.ts

```diff
@@ -7,7 +7,7 @@ import { UserSettingsFeatureContainerComponent } from '../user-settings-feature-
 import { UserSettingsFormComponent } from '../user-settings-form/user-settings-form.component';
 import { UserSettingsDialogComponent } from './user-settings-dialog.component';
 
-class ElectronProviderFake {
+export class ElectronProviderFake {
```

Comment:
I'd not export a class inside a spec file. In order to keep the import space safe with just actual classes. For now i'd just copy this class to the next test.